### ISSUE TITLE
Grab and Drag sluggish when counter is enabled

### ISSUE THREAD

**Issue by sleek22** [mrdokenny/History-AutoDelete#17]
This add-on: https://addons.mozilla.org/en-US/firefox/addon/grab-and-drag/

When show visits on browser icon is enabled, grabbing and dragging becomes delayed and sluggish. The grabbing hand moves a distance before the page moves. Scrolling however, doesn't appear to be affected.

When the show visits option is turned off, the browser becomes responsive again to grabbing and dragging the page.

**Comment by mrdokenny** [comment]
I don't have this performance issue but I can see what's happening that might cripple weaker computers. The addon seems to be creating a page load which is refreshing the counter every time you drag.

Since also that addon is not multiprocess compatible, I think the solution here is to use this e10s compatible addon: https://addons.mozilla.org/en-US/firefox/addon/scroll_anywhere/

**Comment by sleek22** [comment]
Thank you for the quick response.

e10s is out of the question for me as I have about a hundred add-ons installed, most will not be compatible. Also, the add-on you indicated uses the middle mouse button. I'm on a computer from morning to night (full-time investor), my finger wouldn't last a day because the middle mouse button is always stiffer on the mice I've used.

In any event, it's a feature I can live without, for me keeping history in check is the most important thing.

I should add that your add-on is quite amazing. I had installed and tried to use the Expire History By Days add-on, but I and others could not make it work.

Your add-on, on the other hand, started deleting history entries immediately, exactly the function we need.

Thank you for making this wonderful add-on. I truly appreciate it.
